media/js/styleguide/docs/send-to-device.js

```diff
@@ -2,9 +2,16 @@
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+/* eslint-disable no-console */
+
 $(function() {
     'use strict';
 
     var form = new Mozilla.SendToDevice();
+
+    form.geoCallback = function(countryCode) {
+        console.log('The country code is: ' + countryCode);
```

Comment:
Because it's a local dev only page to demonstrate functionality, I left this in.